### PR TITLE
Add Indonesian Locale

### PR DESCRIPTION
**Pull request by sdmoko**
hi @suve do i need to edit the po file header?

### CODE REVIEW

File: lang/id.po [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+"PO-Revision-Date: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"X-Generator: Poedit 1.8.11\n"
+"Last-Translator: Syah Dwi Prihatmoko <sdmoko@sdmoko.net>\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+"Language: id_ID\n"
+
+msgid "msg_help_usage"
+msgstr "Penggunaan: vrms-rpm [options]"
```
Comment (suve) [inline lang/id.po]: It says "options" here, but the manpage uses "opsi".

File: lang/id.po [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+msgstr "%s: opsi tidak diketahui '%s'"
+
+msgid "msg_total_free"
+msgstr "%d paket free"
+
+msgid "msg_total_nonfree"
+msgstr "%d paket non-free"
+
+msgid "msg_rms_happy_ascii"
+msgstr "Hanya paket free - rms akan bangga!                           rms bahagia."
```
Comment (suve) [inline lang/id.po]: Please add a few spaces so this line is exactly 79 characters long.

File: lang/id.po [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+msgid "msg_rms_happy"
+msgstr "Hanya paket free - rms akan bangga!"
+
+msgid "msg_rms_disappoint_ascii"
+msgstr "Lebih dari 10%% paket non-free. Apakah Anda benci kebebasan?              rms kecewa."
```
Comment (suve) [inline lang/id.po]: Remove a few spaces - this line should be 79 characters long.

**Comment by sdmoko** [comment]
@suve please check again, tell me if still wrong. :)
cheers

**Comment by suve** [comment]
Looks fine, merged. Thanks!

**Pull request merged**